src/ai/backend/manager/api/session.py

```diff
@@ -693,6 +694,12 @@ async def create_cluster(request: web.Request, params: dict[str, Any]) -> web.Re
             raise TaskTemplateNotFound
         owner_uuid, group_id, resource_policy = await query_userinfo(request, params, conn)
 
+        enable_sudo_session = await conn.scalar(
+            sa.select([users.c.enable_sudo_session]).where(
+                request["user"]["uuid"] == users.c.uuid,
+            )
+        )
+
```

Comment:
`auth_middleware` [automatically injects](https://github.com/lablup/backend.ai/blob/c2a81534dc7ccc3d70916140d33f05af65788670/src/ai/backend/manager/api/auth.py#L542-L547) most data of the `users` row to the request["user"], including the column you've just added. You can just use it instead of querying twice.